README.md

```diff
@@ -0,0 +1,118 @@
+# pds.api-client
+Federated PDS API which provides actionable end points standardized
```

Comment:
Stylistically, it's customary to put a blank line after heading (and two blank lines before—except at the top of a file).

Reply:
I am ok with that, but this file is automatically generated by the openapi-generator. I removed it from the repository since it will be generated but if we want to update the style we will need to do a new pull request on openapi-generator